lib/OpenQA/BuildResults.pm

```diff
@@ -68,6 +68,8 @@ sub add_review_badge {
     my ($build_res) = @_;
 
     $build_res->{reviewed_all_passed} = $build_res->{passed} == $build_res->{total};
+    $build_res->{reviewed_all_passed_or_softfailed}
+      = $build_res->{passed} + $build_res->{softfailed} == $build_res->{total};
```

Comment:
Think about it as "auto-review" badge. But I can change it if you like. Then I would also change `reviewed_all_passed` for consistency.